### ISSUE TITLE
FoV Rotation control

### ISSUE THREAD

**Issue by simontorres** [cds-astro/aladin-lite#211]
Issue #6 mentions a `setRotation` method but I can't find it. If it was removed would you point me to the reasoning behind it?.

Thanks!.

## Note
If you zoom out a lot and move the field around, the fov comes back rotated.  It would be nice to have control over the rotation of the field , either to return it to no rotation or to rotate it on purpose.

**Comment by tboch** [comment]
pinging @bmatthieu3 : it seems that `setRotation`is no longer available as a method in latest beta. Can you revive this?

**Comment by tboch** [comment]
@simontorres : the method has been renamed `setViewCenter2NorthPoleAngle` , but we will bring back the farmer `setRotation`method name.

**Comment by simontorres** [comment]
Thanks @tboch !

**Comment by bmatthieu3** [comment]
I agree the function label is not that great. Should we authorize setRotation back (without removing setViewCenter2NorthPoleAngle) or define a new label. I think about: setNorthOrientation(angle)

**Comment by tboch** [comment]
`setRotation`sounds fine to me. If you want a better name, I can think of `rotateTo`.
In any case, I'd like `setRotation`to stay there as an alias at least.

**Comment by bmatthieu3** [comment]
I got the setRotation and getRotation back: https://cds-astro.github.io/aladin-lite/Aladin.html#setRotation

**Comment by simontorres** [comment]
Thanks!
